bucket/nimbletext.json

```diff
@@ -1,22 +1,15 @@
 {
-    "version": "3.1.0.17465",
+    "version": "3.1.0.33266",
     "description": "Text manipulation and code generation tool.",
     "homepage": "https://nimbletext.com/",
     "license": "Unknown",
-    "url": "http://nimbletext.com/Download/NimbleText.exe",
+    "url": "https://nimbletext.com/Download/NimbleText.exe",
     "hash": "710ee5e8b449aca2b54c45c426f3e8316da31a78463cae373b40f92a161380f6",
     "bin": "NimbleText.exe",
     "shortcuts": [
         [
             "NimbleText.exe",
             "NimbleText"
         ]
-    ],
-    "checkver": {
-        "url": "http://nimbletext.com/Version/Notes",
-        "regex": "Version ([\\d.]+)"
-    },
-    "autoupdate": {
-        "url": "http://nimbletext.com/Download/NimbleText.exe"
-    }
+    ]
```

Comment:
```suggestion
    ],
    "checkver": {
        "url": "https://nimbletext.com/Version",
        "regex": "([\\d.]+),\\d+",
        "reverse": true
    }
```

Reply:
That URL for checkver is new to me, is it published somewhere by the author? The previous manifest used

https://nimbletext.com/version/notes

Second, BOTH sources of version information list 3.1.0.17465 as the top/bottom of the list, out of order with the actual released file, version 3.1.0.33266.

I'm waiting for a response from the author about what the heck is going on with the version history but the only actual file that you can download is 3.1.0.33266. That's why I suggested removing the version check/autoupdate, at least temporarily.

What am I missing here?

Reply:
NimbleText author here -- yeh this is my bad. I'll let you know when I've fixed it (a lot on my plate at this minute.........)

Incidentally, "https://nimbletext.com/Version" is the better page to use for checking for updates (it's what NimbleText itself uses.)

This mistake hasn't broken nimbletext's check for updates -- by chance -- because the dev build # that was written into that Version file just happened to be a bit smaller number than the released version number.... (ends with .17465, smaller than .33266)  and nimbletext gets the last line from 

    https://nimbletext.com/Version

...and doesn't just check if it's different to itself, but is it a "larger" number.

Huge thanks to Kris for informing me!

Reply:
Also re "That URL for checkver is new to me" -- yes it is undocumented, but if it helps the "scoops" of this world, i'll put a comment on the version/notes page that helps people such as yourself to find the secret best place to find the latest version.

Reply:
@secretGeek Thanks for the quick response! I figured it was probably just a simple mistake.

@Ash258, you've suggested leaving the `checkver` block in place (with the preferred version history URL)...shouldn't it be removed until the version history is properly formatted? I thought it would be safest to remove it and restore the functionality once the author lets us know it's fixed.

Reply:
ok - i think it's fixed, but please check it's good for you there. 
cheers, lb

Reply:
> ok - i think it's fixed, but please check it's good for you there.

Looks good, thanks for the fix!